### ISSUE TITLE
Improve staking page interactiveness

### ISSUE THREAD

**Issue by jak-pan** [galacticcouncil/hydration-ui#1782]
Currently we don't show chart updates on i.e. increasing stake (we should show you what will happen when you increase stake)

**Comment by CJ13th** [comment]
Hey @jak-pan I think this could make a "good first issue" for one of the Polkadot Blockchain Academy alumni to work on during [morekudos.com/carnival](https://www.morekudos.com/carnival). Can we add this to the list?

**Comment by mittal-parth** [comment]
Hey @jak-pan!

Just to make sure, we are on the same page. The '% staked out of total circulating supply' chart should alter when the user is entering the amount to stake, along with the projected APR?

![image](https://github.com/user-attachments/assets/032deab3-a5a6-40f2-a80b-42e0bdc76c0d)

**Comment by jak-pan** [comment]
Hey, no, single users most probably don't have enough stake to push this much, albeit it's ok if we show increase here it will probably don't move much. 

It was meant to work when increasing stake.

It's a little bit complicated as we don't have design for this yet but when increasing stake, you are moving on the time curve as we need to adjust the point in time where you are to not give you more unlocked stake instantly. This should be shown on the chart which you currently don't see as you don't have any stake. Also hovering over individual referenda could show you how much you could move (at least in term of unlock days and %) on the curve based on different voting locks you could chose. End goal is to have voting in app with referendum info that would store your preferred voting amount and lock time and show this movement on the staking curve in all while voting.

**Comment by mittal-parth** [comment]
Thanks for the explanation, looks fairly complex at the outset 😅

**Comment by jak-pan** [comment]
> Thanks for the explanation, looks fairly complex at the outset 😅

Yeah I think it's not the greatest first issue unless you like math pain :) But I will tag a few more issues that are and you can check if you like something.

**Comment by mittal-parth** [comment]
That would be great @jak-pan!
